### PR TITLE
[New Rule] Potential Brute Force Attack Currently in Progress

### PR DESCRIPTION
**Pull request by MakoWish**
## Summary

Since the popularity of working from home, we have had several occasions of team members plugging their company-owned laptops directly into their modems; bypassing their router/firewall protections. Within just a few minutes of this happening, we have observed password spraying / brute forcing attempts to RDP. This rule has been instrumental in helping us to identify this situation and these attacks so we may immediately take corrective actions. 

This rule detects failed logins for multiple user accounts on a single device from the same source address within a short period of time. I have now had this rule in place for about four months, and every hit has been a true positive, so I have set the severity to critical. 

## Contributor checklist

- [X] Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- [X] Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Aegrah** [comment]
This is a nice rule, but I feel like we mostly already cover this behavior in our other detection rules right? For example for Linux we have:
- https://github.com/elastic/detection-rules/blob/main/rules/linux/credential_access_potential_linux_ssh_bruteforce_external.toml
- https://github.com/elastic/detection-rules/blob/main/rules/linux/credential_access_potential_linux_ssh_bruteforce_internal.toml
-https://github.com/elastic/detection-rules/blob/main/rules/linux/credential_access_bruteforce_password_guessing.toml

And for Windows:
- https://github.com/elastic/detection-rules/blob/6d7df50d78906a43143274152dcbbf7946671858/rules/windows/credential_access_bruteforce_admin_account.toml
- https://github.com/elastic/detection-rules/blob/6d7df50d78906a43143274152dcbbf7946671858/rules/windows/credential_access_bruteforce_multiple_logon_failure_same_srcip.toml
- https://github.com/elastic/detection-rules/blob/6d7df50d78906a43143274152dcbbf7946671858/rules/windows/credential_access_bruteforce_multiple_logon_failure_followed_by_success.toml

And MacOS:
- https://github.com/elastic/detection-rules/blob/6d7df50d78906a43143274152dcbbf7946671858/rules/macos/credential_access_potential_macos_ssh_bruteforce.toml

Basically covering this rule. Correct me if I'm wrong and this rule detects something that's not listed in the rules above^. 

Do we have specific reasoning why we would add an additional cross platform threshold rule?

A rule such as this one (any brute force rule that targets external-facing hosts) are also inherently very very very noisy, because if there is an IP available on the internet to whom anyone may connect, you can be sure there will be thousands of people brute forcing it the moment it gets scraped and becomes available on Shodan. 

Just as reference, our Linux external brute forcing rule generated 52,535 alerts over the last 30 days. I think having this threshold rule will bump up the alerts even further.

**Comment by Aegrah** [comment]
Note: Had this rule running as a test in my environment, and it seems to trigger around 40 times per hour (in my public-facing Windows stack). At the same time, two other brute-forcing rules trigger the same behavior. Currently, I think it would make more sense to clean our current brute-forcing rules, rather than implement a new one. WDYT @w0rk3r?

**Comment by w0rk3r** [comment]
Hey @MakoWish, I appreciate the effort behind this rule. However, it overlaps a lot with the existing rules, which will cause duplicated alerts. Considering we're limiting to `event.action:(log_on or logon-failed or ssh_login)`, I don't think we are getting enough additional coverage to justify the potential noise we add, and if we removed this, the rule would be too noisy.

Also, as the noise of this kind of rule depends on the environment specifics, and can require quite some tuning, so this may be better suited as a local rule atm.

**Comment by MakoWish** [comment]
Okay. This is a **_very_** quiet rule in our environment, so any alert is an immediate action on my part. IMO, it is also much cleaner than the other overlapping rules, so I will keep this running locally with the others disabled.
